docs/components/rule.md

```diff
@@ -323,6 +323,11 @@ Flags:
                                  from other components.
       --grpc-grace-period=2m     Time to wait after an interrupt received for
                                  GRPC Server.
+      --grpc-query-endpoint=<endpoint> ...
```

Comment:
Naturally, there will be a question which should be preferred. AFAICT there's no difference right now functionally. Also, in the code we start with HTTP and then gRPC. Could we add a sentence about these two points to the docs?

Reply:
Sry was on vacation and missed that. I will add details about it to the docs.

Reply:
> Naturally, there will be a question which should be preferred. AFAICT there's no difference right now functionally. Also, in the code we start with HTTP and then gRPC. Could we add a sentence about these two points to the docs?

Recording rules that produce native histograms are only supported via gRPC, since HTTP does not support native histograms and will return 0. I added documentation about that and http taking priority over gRPC in "HA groups".

Reply:
Maybe gRPC should become the default going forward, and HTTP can be used for running rules against external services.